src/index.html

```diff
@@ -12,6 +11,163 @@
   </head>
   <body>
     <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+
+      <fieldset>
+        <legend>Personal information</legend>
+        <div>
+          <label for="in1">Surname:</label>
+          <input type="text"
+                 id="in1"
+                 autocomplete="off"
+                 name="surname"
+          >
```

Comment:
```suggestion
          <input 
            type="text"
            id="in1"
            autocomplete="off"
            name="surname"
          >
```
recheck code style with guideline everywhere
https://mate-academy.github.io/style-guides/htmlcss.html